01_stratified_classification/nbconverted/plot_univariate_f_dists.py

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Visualization of f-statistic distributions for selected features
+
+# In[1]:
+
+
+import os
+import sys
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+import pancancer_evaluation.config as cfg
+import pancancer_evaluation.utilities.analysis_utilities as au
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+
+# ### Load selected coefficients
+
+# In[2]:
+
+
+results_dir = os.path.join('results', 'univariate_fs', 'pancancer')
+
+
+# In[3]:
+
+
+# gene to analyze features for
+gene = 'PIK3CA'
+
+id_coefs_info = []
+for identifier, coefs_list in au.generate_nonzero_coefficients_fs(
+        results_dir, cfg.fs_methods):
+    if not identifier.startswith(gene): continue
+    for fold_no, coefs in enumerate(coefs_list):
+        id_coefs_info.append([identifier, fold_no, coefs])
+        
+print(len(id_coefs_info))
+
+
+# In[4]:
+
+
+print(len(id_coefs_info[0]))
+print(id_coefs_info[0])
```

Comment:
Maybe a quick comment about what someone should take from this? Just that it's properly formatted?